typings/index.d.ts

```diff
@@ -0,0 +1,13 @@
+declare global {
+    const Translator: {
```

Comment:
Here we have type definitions of global variables so we have type checking for them as well. Without these type definitions TS shows errors. 
We can declare more types (as for `Translator`) or less (as for `eZ` below).